lib/foreman_maintain/hammer.rb

```diff
@@ -0,0 +1,37 @@
+module ForemanMaintain
+  class Hammer
```

Comment:
Yes, I have added hammer as a separate feature using which we can have 3 methods as below

1. password_available?
2. username_available?
3. main_command which will return hammer command string with -u & -p  options.

Should I move logic of retrieving of username & password from this file into feature itself?